webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -271,15 +269,6 @@ const useModalExperimentMove = ({ onClose }: Props): ModalHooks => {
         ),
         message: 'Move Success',
       });
-      if (sourceProjectId) {
-        const newPinned = { ...projectSettings.pinned };
-        const pinSet = new Set(newPinned[sourceProjectId]);
-        for (const experimentId of experimentIds) {
-          pinSet.delete(experimentId);
-        }
-        newPinned[sourceProjectId] = Array.from(pinSet);
-        updateProjectSettings({ pinned: newPinned });
-      }
```

Comment:
it looked like doing `useSettings` in mutliple places was causing some ping-ponging, and we are already filtering by whether pinned experiments are in the project, so we don't strictly speaking need this (and this doesn't fully solve the problem anyways-- consider other people moving the experiments you pinned, etc.).